### ISSUE TITLE
Not finishing color stamp will get the program stuck.

### ISSUE THREAD

**Issue by mineTomek** [mineTomek/GreatConsole#1]
## Example:
```csharp
GreatConsole.ConsoleColors.WriteFromString("&g Hi!");
//                                            ^ No " ' "

GreatConsole.ConsoleColors.WriteLine(
    "Hello", 
    GreatConsole.ConsoleColors.AllowedColor.Red
); // This will not execute :(
```
The program will be stuck on the first line.

**Comment by mineTomek** [comment]
Wow! It's a bizarre bug!

**Comment by mineTomek** [comment]
Fixed with #3 pull request
